core/src/optimistic_confirmation_verifier.rs

```diff
@@ -175,7 +203,9 @@ mod test {
         {
             let blockstore = Blockstore::open(&blockstore_path).unwrap();
```

Comment:
Same as above while you're already in here - use the other macro that will automatically delete itself

Reply:
updated